meson.build

```diff
@@ -21,12 +21,16 @@ config_file = configure_file(
     configuration: config_data
 )
 
+subdir('data')
+subdir('po')
+
 executable(
     meson.project_name(),
     'src/Application.vala',
     'src/MainWindow.vala',
     'src/ShortcutLabel.vala',
     'src/Views/ShortcutsView.vala',
+    gresource,
     config_file,
     dependencies: [
         dependency('glib-2.0'),
```

Comment:
I couldn't comment on the actual line, but we now require granite-7 >= 7.3.0 because of automatic load of Application.css:

https://github.com/elementary/granite/commit/2b6d52043cb1f8a30e54675f8c375f2712ad5c92

We should explicit it in the meson.build and README.